internal/controller/controllers/bmh_agent_controller.go

```diff
@@ -553,27 +554,6 @@ func (r *BMACReconciler) ensureBMHDetached(log logrus.FieldLogger, bmh *bmh_v1al
 	return reconcileComplete{dirty: true, stop: true}
 }
 
-// The detached annotation is added if the installation of the agent associated with
```

Comment:
Actually I think it is better to keep they were before this change and just rename and change the function **addBMHDetachedAnnotationIfAgentHasStartedInstallation**.

Reply:
Ok sounds good! I reverted it back but I'm not sure what the name of the function should be 😅 any ideas?

Reply:
addBMHDetachedAnnotationIfHostIsRebooting ?